EPRN.Portal/Services/HomeServices/BaseHomeService.cs

```diff
@@ -11,6 +11,9 @@ public abstract class BaseHomeService : IHomeService
 
         public BaseHomeService(IOptions<AppConfigSettings> configSettings)
         {
+            if (configSettings == null)
+                return;
+
```

Comment:
Don't need these lines. The issue over the loading of the config has been resolved.

Reply:
Done.